dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditReplayThread.java

```diff
@@ -99,6 +101,16 @@ void drainCounters(Mapper.Context context) {
     }
   }
 
+  void drainCommandLatencies(Mapper.Context context) throws IOException {
```

Comment:
We shouldn't catch and swallow `InterruptedException` here. It should bubble up to the `cleanup()`. Also, no need to re-throw the IOE.